Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1646,13 +1695,17 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'GenSim',
         'Digi',
         'Reco',
+#        'RecoFakeHLT',
         'HARVEST',
+        'HARVESTFakeHLT',
         'ALCA',
     ],
     PU = [
         'Digi',
         'Reco',
+#        'RecoFakeHLT',
```

Comment:
These two lines are commented on purpose. If uncommented, the process modifier `run2_HECollapse_2018` will be used in step3 (RECO+DQM) of the 2018 wfs `*.6`, but this will lead to wf `10824.6` (and maybe others) failing with [1].

Currently, `run2_HECollapse_2018` is not used in step3 of these wfs, but I'm not sure if that is on purpose. Since this is beyond the scope of this PR, these wfs are effectively left unchanged (i.e. working, because `run2_HECollapse_2018` is not applied in step3, both pre- and post-PR).

FYI: @cms-sw/alca-l2 @cms-sw/hcal-dpg-l2

[1]
```
----- Begin Fatal Exception 30-Apr-2023 17:35:02 CEST-----------------------
An exception of category 'Conditions mismatch' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'dqmofflineOnPAT_1_step'
   [2] Prefetching for module SingleTopTChannelLeptonDQM_miniAOD/'singleTopElectronMediumDQM_miniAOD'
   [3] Prefetching for module PATMuonSlimmer/'slimmedMuons'
   [4] Prefetching for module PATMuonSelector/'selectedPatMuons'
   [5] Prefetching for module PATMuonProducer/'patMuons'
   [6] Prefetching for module MuonProducer/'muons'
   [7] Prefetching for module MuonIdProducer/'muons1stStep'
   [8] Prefetching for module HBHEIsolatedNoiseReflagger/'hbhereco@cpu'
   [9] Prefetching for module TrackExtrapolator/'trackExtrapolator'
   [10] Prefetching for module DuplicateListMerger/'generalTracks'
   [11] Prefetching for module TrackProducer/'mergedDuplicateTracks'
   [12] Prefetching for module DuplicateTrackMerger/'duplicateTrackCandidates'
   [13] Prefetching for module TrackCollectionMerger/'preDuplicateMergingGeneralTracks'
   [14] Prefetching for module TrackCollectionMerger/'earlyGeneralTracks'
   [15] Prefetching for module TrackProducer/'highPtTripletStepTracks'
   [16] Prefetching for module MkFitOutputConverter/'highPtTripletStepTrackCandidates'
   [17] Prefetching for module MkFitSeedConverter/'highPtTripletStepTrackCandidatesMkFitSeeds'
   [18] Prefetching for module SeedCreatorFromRegionConsecutiveHitsEDProducer/'highPtTripletStepSeeds'
   [19] Prefetching for module CAHitTripletEDProducer/'highPtTripletStepHitTriplets'
   [20] Prefetching for module HitPairEDProducer/'highPtTripletStepHitDoublets'
   [21] Prefetching for module SeedingLayersEDProducer/'highPtTripletStepSeedLayers'
   [22] Prefetching for module TrackClusterRemover/'highPtTripletStepClusters'
   [23] Prefetching for module TrackProducer/'lowPtQuadStepTracks'
   [24] Prefetching for module CkfTrackCandidateMaker/'lowPtQuadStepTrackCandidates'
   [25] Prefetching for module SeedCreatorFromRegionConsecutiveHitsEDProducer/'lowPtQuadStepSeeds'
   [26] Prefetching for module CAHitQuadrupletEDProducer/'lowPtQuadStepHitQuadruplets'
   [27] Prefetching for module HitPairEDProducer/'lowPtQuadStepHitDoublets'
   [28] Prefetching for module SeedingLayersEDProducer/'lowPtQuadStepSeedLayers'
   [29] Prefetching for module TrackClusterRemover/'lowPtQuadStepClusters'
   [30] Prefetching for module TrackTfClassifier/'initialStep'
   [31] Prefetching for module RecoChargedRefCandidatePrimaryVertexSorter/'firstStepPrimaryVertices'
   [32] Prefetching for module FastjetJetProducer/'ak4CaloJetsForTrk'
   [33] Calling method for module CaloTowersCreator/'caloTowerForTrk'
Exception Message:
Requested conditions of type HcalChannelQuality for cell (0x45304007) (HE -16,7,3) got conditions for cell (0x0) 
----- End Fatal Exception -------------------------------------------------
```

Reply:
@missirol  
it I remember it right,  for wfs like 10824.6 ("collapsed HE for HLT") the idea was to apply the "HE collapsing" [1]  only to HLT, not to Offline reco.  This is how it was used in 2018 data taking.   

But I must admit I don't quite understand why adding `run2_HECollapse_2018` modifier to step3 of these wfs can cause the fail. I'd expect  `hcalLocalReco` should be working the same way for HLT and Offline, means first to produce **full** set of RecHits (uncollapsed) using full set of conditions entries. And only after it to collapse (and redefine) certain HE depths for using them in CaloTowers formatting (also customized)...  

While the fatal exception shows that non-customized (non-affected by modifier) CaloTower maker uses as an input collapsed (affected by modifier) HE RecHits...  

----------------------------------------------------
[1]
https://github.com/cms-sw/cmssw/pull/22001